src/app/backend/resource/job/detail_test.go

```diff
@@ -100,3 +101,60 @@ func TestGetJobDetail(t *testing.T) {
 		}
 	}
 }
+
+func TestDeleteJob(t *testing.T) {
+	var jobCompletions int32
+	var parallelism int32
+
+	cases := []struct {
+		namespace, name string
+		expectedActions []string
+		job             *batch.Job
+	}{
+		{
+			"ns-1", "job-1",
+			[]string{"get", "list", "delete"},
+			createJob("job-1", "ns-1", map[string]string{"app": "test"}),
+		},
+		{
+			namespace: "ns-1", name: "job-1",
+			expectedActions: []string{"get", "list", "delete", "delete"},
+			job: &batch.Job{
+				ObjectMeta: metaV1.ObjectMeta{
+					Name: "ns-1", Namespace: "job-1", Labels: map[string]string{"app": "test"},
+				},
+				Spec: batch.JobSpec{
+					Selector:    &metaV1.LabelSelector{MatchLabels: map[string]string{"app": "test"}},
+					Completions: &jobCompletions,
+					Parallelism: &parallelism,
+					Template: v1.PodTemplateSpec{
+						ObjectMeta: metaV1.ObjectMeta{
+							Name:      "test-pod",
+							Namespace: "ns-1",
+							Labels:    map[string]string{"app": "test"},
+						},
+					},
+				},
+			},
+		},
```

Comment:
Yes but second test will probably not work like this. You need to create pod list and pass it to `fake.NewSimpleClientset` method. Then when `client.Pods().List()` is called it will return pod list passed to this method.